sympy/stats/tests/test_continuous_rv.py

```diff
@@ -1497,3 +1497,15 @@ def test_ContinuousDistributionHandmade():
     assert median(space.value) == Interval(1, 2)
     assert E(space.value) == Rational(3, 2)
     assert variance(space.value) == Rational(13, 12)
+
+def test_issue_16318():
+    #test compute_expectation function of the SingleContinuousDomain
+    x,y = symbols('x y')
+    N = SingleContinuousDomain(x, Interval(0, 1))
+    raises (ValueError, lambda: SingleContinuousDomain.compute_expectation(N, x+1, {x, y}))
+    #test quantile function of the NormalDistribution
+    mu = Symbol("mu")
+    sigma = Symbol("sigma", positive=True)
+    p = Symbol("p")
+    X = Normal("x", mu, sigma)
+    assert mu + sqrt(2)*sigma*erfinv(2*p - 1) == quantile(X)(p)
```

Comment:
This one is not required as it is already tested in `def test_multiple_normal():`. We don't need multiple tests asserting the same thing.

Reply:
Yes you're right! I didn't notice because the "quantile" function appears to not be covered on Codecov. I removed this section. Thank you very much for the help!